commit 6fa56caa95ed30b7de4a7a5bd267070e213d0a0f
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Tue May 3 21:53:29 2022 +0900

    start.sh

diff --git a/scripts b/scripts
new file mode 120000
index 0000000..a339954
--- /dev/null
+++ b/scripts
@@ -0,0 +1 @@
+../scripts
\ No newline at end of file
diff --git a/start.sh b/start.sh
new file mode 100755
index 0000000..e098414
--- /dev/null
+++ b/start.sh
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+(cd server && npm run server-start &)
+SERVER_PID="$!"
+trap 'scripts/kill-all-child-pids.sh "$SERVER_PID"' 2 # 2 means SIGINT = Ctrl + c
+
+(cd client && npm run client-start &)
+CLIENT_PID="$!"
+trap 'scripts/kill-all-child-pids.sh "$CLIENT_PID"' 2 # 2 means SIGINT = Ctrl + c
+
+(cd file-server && npm run file-server-start &)
+FILE_SERVER_PID="$!"
+trap 'scripts/kill-all-child-pids.sh "$FILE_SERVER_PID"' 2 # 2 means SIGINT = Ctrl + c
+
+# wait infinitely
+tail -f /dev/null
\ No newline at end of file
